extension-readme.md

```diff
@@ -196,25 +196,30 @@ For full details, see [Working with AWS Step Functions](https://docs.aws.amazon.
+| **AWS: Sign out**                                      | Disconnects the Toolkit from the currently-connected AWS account.                                                                                                                                                                                                                                              |
+| **AWS: Submit Quick Feedback...**                      | Submit a private, one-way message and sentiment to the AWS Toolkit dev team. For larger issues that warrant conversations or bugfixes, please submit an issue in Github with the **AWS: Create a New Issue on Github** command.                                                                                |
+| **AWS: View AWS Toolkit Logs**                         | Displays log files that contain general Toolkit diagnostic information.                                                                                                                                                                                                                                        |
+| **AWS: View Quick Start**                              | Open this quick-start guide.                                                                                                                                                                                                                                                                                   |
+| **AWS: View CDK Documentation**                        | Opens the [user guide](https://docs.aws.amazon.com/console/toolkit-for-vscode/aws-cdk-apps) for the CDK portion of the Toolkit.                                                                                                                                                                                |
+| **AWS: View Toolkit Documentation**                    | Opens the [user guide](https://docs.aws.amazon.com/console/toolkit-for-vscode/welcome) for the Toolkit.                                                                                                                                                                                                        |
+| **AWS: View Source on GitHub**                         | Opens the [GitHub repository](https://github.com/aws/aws-toolkit-vscode) for the Toolkit.                                                                                                                                                                                                                      |
+| **Explorer: Focus on AWS CDK Explorer (Preview) View** | Opens the **Explorer** Side Bar and expands the **AWS CDK Explorer (Preview)** View.                                                                                                                                                                                                                           |
```

Comment:
If this one is provided by VS Code, I'd suggest removing it from the list, since a future version of VS Code could change the command wording, or even remove the command.

Reply:
In that case, we should also remove https://github.com/aws/aws-toolkit-vscode/pull/1121/files#diff-e420a363d17c376f750fb6055914d6a4R209 , this also appears to be provided by VS Code (`AWS:` being the panel name)